src/Microsoft.PowerShell.Commands.Utility/commands/utility/ImportAliasCommand.cs

```diff
@@ -99,6 +100,93 @@ public SwitchParameter Force
         #endregion Parameters
 
         #region Command code
+        
+        private static bool OnlyContainsWhitespace(string line)
+        {
+            bool result = true;
+
+            foreach (char c in line)
+            {
+                if (char.IsWhiteSpace(c) && c != '\n' && c != '\r')
```

Comment:
Now we use StreamReader.Readline() - it skip EOL chars.

And now we can replace the method with ReadOnlySpan<char>.isWhitespace()
https://source.dot.net/#System.Private.CoreLib/shared/System/MemoryExtensions.cs,28

Reply:
Just to make sure, you want me to use a reader instead of the `for`? Since you previously said

>I think StringReader complicates the code and doesn't add readability. Using StringReader looks like personal preference.

Reply:
My current comment is only that make no sense to check `c != '\n' && c != '\r'`.
Previous comment was that we could not use StreamReader at all.

Reply:
clear, sorry for the confusion.